### PR TITLE
Fixed mobile layout for blog post newsletter signup form

### PR DESCRIPTION
**Pull request by mmmavis**
No functionality changes are introduced in this PR. This is mainly to adjust mobile layout for the sign up module.

image

Link to sample test page: https://foundation-s-11839-blog-76rjtd.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
Related PRs/issues: #11839

### CODE REVIEW

File: source/js/components/newsletter-signup/organisms/form-specific-style.js [hunk 1]
```diff
@@ -93,11 +93,10 @@ export const FORM_STYLE = {
     headingClass: "tw-h5-heading",
   },
   "mozfest-callout-box": {
+    descriptionClass: `medium:tw-w-4/5`,
     fieldStyle: "filled",
     headingLevel: 3,
     headingClass: `tw-h3-heading medium:tw-w-4/5`,
-    descriptionClass: `medium:tw-w-4/5`,
-    headingClass: "tw-h3-heading",
```
Comment (mmmavis) [inline source/js/components/newsletter-signup/organisms/form-specific-style.js:100]: removed dupe key `headingClass`

File: source/js/components/newsletter-signup/organisms/form-specific-style.js [hunk 2]
```diff
@@ -93,11 +93,10 @@ export const FORM_STYLE = {
     headingClass: "tw-h5-heading",
   },
   "mozfest-callout-box": {
+    descriptionClass: `medium:tw-w-4/5`,
     fieldStyle: "filled",
     headingLevel: 3,
     headingClass: `tw-h3-heading medium:tw-w-4/5`,
-    descriptionClass: `medium:tw-w-4/5`,
```
Comment (mmmavis) [inline source/js/components/newsletter-signup/organisms/form-specific-style.js:99]: moved to line 96

**Review by kristinashu — APPROVED** [review]
Much better, thank you for the quick fix!

**Review by danielfmiranda — APPROVED** [review]
Looks great! Thanks @mmmavis

**Pull request merged**